### PR TITLE
Add Fan Art to snatchSelection.mako

### PR DESCRIPTION
**Pull request by Thraxis**
- [x] Select Columns that can not be hidden
- [x] Add sorting on select columns?  (optional)
- [x] Re-initialize the table every refresh or the search needs to be a proper ajax load of just table data (without it, while refreshing it shows the hidden columns) @p0psicles 

image

### CODE REVIEW

**Comment by fernandog** [comment]
@Thraxis is it possible to make "Season" font black?

light theme:
image

**Comment by fernandog** [comment]
@Thraxis can you rebase please?

**Comment by fernandog** [comment]
Approved

**Comment by Thraxis** [comment]
interesting fact... saveSort is enabled, but the query to load the search results over-rides it anyway.  But there is a reset sort button now and it will reset the sort.

**Comment by fernandog** [comment]
Nice! Will test it!

**Comment by fernandog** [comment]
Working nice!

**Comment by fernandog** [comment]
@duramato

**Comment by fernandog** [comment]
@Thraxis are you having this issue too? maybe because of the refreshing

image

**Comment by Thraxis** [comment]
The refresh issue I had fixed in my testing branch but forgot to fix in the PR branch...   widths should be good now with this fix as well

**Comment by fernandog** [comment]
@Thraxis do you know how to fix the last TODO (PR body) ? What about this codacy-bot issues?

**Comment by Thraxis** [comment]
The change to use ajax for only refreshing the table will probably need to be future work.  The current code will re-initialize the tablesorter as soon as it can.

As for the codacy message, it's not an error so much as a "If you're doing this your CSS is getting too convoluted you should refactor it"

**Comment by Thraxis** [comment]
Fixed a type-o that prevented the refresh in the last update... squashed to one commit and rebased to current development

**Comment by fernandog** [comment]
@thraxis your PRs will be merged after ui-rebased

@medario

File: gui/slick/views/snatchSelection.mako [hunk 1]
```diff
@@ -98,7 +103,7 @@
                 % endif
             </ul>
         </div><!-- #tags //-->
-        <div id="summary">
+        <div id="summary" ${"class=\"summaryFanArt\"" if bool(sickbeard.FANART_BACKGROUND) else ""}>
```
Comment (fernandog) [inline gui/slick/views/snatchSelection.mako]: @Thraxis need to remove all the `bool`. apart from that is there anything else that needs to be done?

Comment (Thraxis) [inline gui/slick/views/snatchSelection.mako]: Nope that should be all I have left to fix on this

**Comment by fernandog** [comment]
Approved

**Comment by fernandog** [comment]
@thraxis thanks!

If you want to re-work the AJAX feel free.

**Comment by fernandog** [comment]
@Thraxis are you going to work in the snatchSelection AJAX fix?

**Pull request merged**